x-pack/plugin/ml-feature-index-builder/src/main/java/org/elasticsearch/xpack/ml/featureindexbuilder/rest/action/RestGetDataFrameJobsAction.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.ml.featureindexbuilder.rest.action;
+
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.common.ParseField;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.rest.BaseRestHandler;
+import org.elasticsearch.rest.RestController;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.action.RestToXContentListener;
+import org.elasticsearch.xpack.ml.featureindexbuilder.FeatureIndexBuilder;
+import org.elasticsearch.xpack.ml.featureindexbuilder.action.GetDataFrameJobsAction;
+
+public class RestGetDataFrameJobsAction extends BaseRestHandler {
+    public static final ParseField ID = new ParseField("id");
+
+    public RestGetDataFrameJobsAction(Settings settings, RestController controller) {
+        super(settings);
+        controller.registerHandler(RestRequest.Method.GET, FeatureIndexBuilder.BASE_PATH + "job/{id}/", this);
```

Comment:
Should `job` be changed to the plural `jobs`?  In ML we were told to use the plurals of `anomaly_detectors` and `datafeeds`.  Other APIs that return lists of configs are also plural - `nodes`, `indices`, `aliases`, etc.